### PR TITLE
Trace insurance changes: show changes on client

### PR DESCRIPTION
**Pull request by dkaplunov-fd**

### CODE REVIEW

File: src/components/chat/tabs/insurance-trace/InsuranceTrace.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { getFieldName } from 'lib/helpers'
+import * as React from 'react'
+import { Table } from 'semantic-ui-react'
+
+const InsuranceTrace = ({traceData}) => {
+  if (!traceData || traceData.length === 0) return ''
+  return (
+  <React.Fragment>
```
Comment (Iamsamwhoami) [inline src/components/chat/tabs/insurance-trace/InsuranceTrace.tsx]: Ts supports the fragment syntax so you can do `<></>` a little bit neater :)

Comment (dkaplunov-fd) [reply src/components/chat/tabs/insurance-trace/InsuranceTrace.tsx]: Done

File: src/components/chat/tabs/insurance-trace/InsuranceTrace.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+          <Table.Row key={trace.id}>
+            <Table.Cell>{getFieldName(trace.fieldName)}</Table.Cell>
+            <Table.Cell>{trace.date}</Table.Cell>
+            <Table.Cell>{trace.oldValue}</Table.Cell>
+            <Table.Cell>{trace.newValue}</Table.Cell>
+            <Table.Cell>{trace.userId}</Table.Cell>
+          </Table.Row>
+        ))}
+      </Table>
+    </React.Fragment>
```
[the remark this answers is not in the document]
Comment (dkaplunov-fd) [reply src/components/chat/tabs/insurance-trace/InsuranceTrace.tsx]: Fixed, sorry for misunderstanding:), now is Ok?

File: src/components/chat/tabs/insurance-trace/InsuranceTrace.tsx [hunk 3]
```diff
@@ -4,9 +4,9 @@ import { Table } from 'semantic-ui-react'
 
 const InsuranceTrace = ({ traceData }) =>
   !traceData || traceData.length === 0 ? (
-    <div />
+    <></>
```
Comment (hedvigoscar) [inline src/components/chat/tabs/insurance-trace/InsuranceTrace.tsx]: You can possibly return `null` here if you want to represent the case of 'no return'

Comment (dkaplunov-fd) [reply src/components/chat/tabs/insurance-trace/InsuranceTrace.tsx]: Done

**Pull request merged**